docs/codeql/codeql-cli/creating-codeql-databases.rst

```diff
@@ -210,11 +210,10 @@ commands that you can specify for compiled languages.
 
 - C# project built using ``dotnet build``::
 
-     For C# projects using either `dotnet build` or `msbuild`, you should specify `/p:UseSharedCompilation=false`
-     in the build command. It is also a good idea to add `/t:rebuild` to ensure that all code will be built (code
-     that is not built will not be included in the CodeQL database):
+     It is a good idea to add `/t:rebuild` to ensure that all code will be built, or do a
+     prior `dotnet clean` (code that is not built will not be included in the CodeQL database):
 
-     codeql database create csharp-database --language=csharp --command='dotnet build /p:UseSharedCompilation=false /t:rebuild' 
+     codeql database create csharp-database --language=csharp --command='dotnet build /t:rebuild'
 
```

Comment:
This change is not directly related to the packaging change. I am not sure if it should be backported.

Reply:
@criemen can you advise?

Reply:
@hvitved as far as I'm aware, the code changes for this docs change is in `rc/3.7`, and therefore, we should also backport the docs changes. Can you please confirm that this is correct?

Reply:
Yes, should be ok to backport.